.gitignore

```diff
@@ -0,0 +1,2 @@
+/go.sum
```

Comment:
We shoudn't ignore go.sum https://github.com/golang/go/wiki/Modules#releasing-modules-all-versions

Reply:
Good spot, thanks. Had assumed generated == don't commit, but it seems not.

We'll need to be careful to update and commit this along with the changes which caused it. Some kind of automation would be helpful, or I suspect this will be frequently overlooked.

Reply:
I found something like this https://github.com/renovatebot/renovate or we can try to do something with git-hooks and 'go mod tidy'